pg_metadata/test/test_sql.py

```diff
@@ -25,11 +25,17 @@ def _insert(self, feature_map, table='dataset', return_value=None):
 
     def test_html_template(self):
         """ Test HTML template. """
+        theme_feature = {
+            'code': "'A01'",
+            'label': "'test theme'",
+        }
+        self._insert(theme_feature, 'theme')
         dataset_feature = {
             'table_name': "'lines'",
             'schema_name': "'pgmetadata'",
             'title': "'Test title'",
             'abstract': "'Test abstract.'",
+            'themes': "'{\"A01\"}'",
```

Comment:
You must add the themes in the template as well

Reply:
that is to say ?

Reply:
I want the test to cover the HTML rendering, with 2 themes.
You just added the themes field, but you are not using it in the test.
That's why the CI is passing. You must add the themes in the template used for testing a few lines below and also update the HTML output showing your themes with the label.